### PR TITLE
adapt plot to explore page

### PR DESCRIPTION
**Pull request by alisman**
Add new plots tab

TODO:
- [ ] Issue for cases/samples w/o demographics

### CODE REVIEW

**Comment by inodb** [comment]
Need to make sure file count lines up with humantumoratlas.org:

image
vs
image

File: lib/helpers.ts [hunk 1]
```diff
@@ -172,6 +174,11 @@ export function fillInEntities(data: LoadDataResult): Entity[] {
             specimen.ParticipantID || (parentIdMatch ? parentIdMatch[0] : '');
     });
 
+    // get rid of files
+    data.files = data.files.filter((f) => {
```
Comment (alisman) [inline lib/helpers.ts:178]: we can't do this.  we must deal with these as legitimate

File: next.config.js [hunk 2]
```diff
@@ -14,4 +14,7 @@ module.exports = withMDX({
             },
         ];
     },
+    typescript: {
```
Comment (alisman) [inline next.config.js:17]: fix this

**Comment by alisman** [comment]
@onursumer that sounds good. i can't remember though what the result of this problem is in the interface.

**Pull request merged**